### ISSUE TITLE
Documentation could be more complete for newbies but thank you!

### ISSUE THREAD

**Issue by marcfilleul** [robiveli/js-captcha#16]
Hi,

Just here to thank you!
I've been looking for a simple and good captcha solution for years and it looks like I've finally foud it ;)

I've struggled to complete the setup so I think the documentation could be more complete but I'm happy.

The main pain point was how to submit the form after success validation.
I couldn't find this info in the docs.

Thanks again!

**Comment by robiveli** [comment]
Thank you for feedback :)

You could easily use [demo](https://www.rvdizajn.com/js-captcha/) source code with `callback` function modified for form submit, for example:

```
callback: function (response, $captchaInputElement) {
    if (response == 'success') {
        // form submit continue, e.g:
        fetch('/url').then(function(response) {
            console.log(response);
        });
    } 
}
```

**Comment by marcfilleul** [comment]
Thanks.
Yes, I've figured it out but as a JS newbie, I had some trouble understanding what I could or couldn't do.
Because your library is likely to be used in a real contact form situation, I suppose it could help to have the info in the docs.

When searching for a solution, I've found that we usually put the `e.preventDefault();` in the error response callback.
But it wasn't working here so I've looked for a simple solution to effectively submit the form if the response is success.

I've finally found this simple callback: `form.submit();`

Adding this one as an example in your docs would help newbies a lot I think ;)

Best regards.
